dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ValidationIT.java

```diff
@@ -168,12 +168,12 @@ public void testWdlWorkflow() {
         workflow = workflowsApi.refresh(workflow.getId());
         Assert.assertFalse(isWorkflowVersionValid(workflow, "master"));
 
-        // change to valid tool - should be valid
+        // change to valid tool - should be invalid
         workflow.setWorkflowPath("/validTool.wdl");
         workflow = workflowsApi.updateWorkflow(workflow.getId(), workflow);
         workflowsApi.updateWorkflowPath(workflow.getId(), workflow);
         workflow = workflowsApi.refresh(workflow.getId());
-        Assert.assertTrue("Should be valid", isWorkflowVersionValid(workflow, "master"));
+        Assert.assertFalse("Should be invalid", isWorkflowVersionValid(workflow, "master"));
```

Comment:
Reverted the test back to what it was originally because WDL tools with one task call can be registered as a WDL workflow (based on discussion with Denis).